simexpal/instances.py

```diff
@@ -22,6 +22,12 @@ class DownloadException(Exception):
 def download_instance(inst_yml, instances_dir, filename, partial_path, ext):
 	import requests
 
+	def download_as_stream(url, download_path):
+		with requests.get(url, stream=True) as request:
+			with open(download_path, 'wb') as f:
+				for chunk in request.iter_content(chunk_size=4096):
```

Comment:
4096 is rather small. I think it would be fine to reserve 64KiB or even a megabyte or so.

Reply:
I changed it to 1MB (``chunk_size=1000000``)